src/components/RadioButtonField/RadioButtonField.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ButtonInput, ButtonLabel, RadioButton, RadioButtonWrap } from './styles';
+
+const RadioButtonField = ({ inputs, active, onChange }) => {
+  const handleInputChange = (e) => {
+    onChange(e.target.value);
+  };
+  return (
+    <RadioButtonWrap>
+      {inputs.map((input) => (
+        <RadioButton key={input.key}>
```

Comment:
You should use `input.id` as a key. `input.value` as a value and `input.label` as a text for radio.